service/integration/resource_mappings_test.go

```diff
@@ -46,6 +66,226 @@ func (s *ResourceMappingsSuite) getResourceMappingFixtures() []fixtures.FixtureD
 	}
 }
 
+/*
+ Resource Mapping Groups
+*/
+
+func (s *ResourceMappingsSuite) Test_ListResourceMappingGroups() {
+	testData := s.getResourceMappingGroupFixtures()
+	rmGroups, err := s.db.PolicyClient.ListResourceMappingGroups(s.ctx, &resourcemapping.ListResourceMappingGroupsRequest{})
+	s.Require().NoError(err)
+	s.NotNil(rmGroups)
+	for _, testRmGroup := range testData {
+		found := false
+		for _, rmGroup := range rmGroups {
+			if testRmGroup.ID == rmGroup.GetId() {
+				found = true
+				break
+			}
+		}
+		s.True(found, fmt.Sprintf("expected to find resource mapping group %s", testRmGroup.ID))
+	}
+}
+
+func (s *ResourceMappingsSuite) Test_ListResourceMappingGroupsWithNamespaceIdSucceeds() {
+	scenarioDotComRmGroup := s.f.GetResourceMappingGroupKey("scenario.com_ns_group_1")
+	rmGroups, err := s.db.PolicyClient.ListResourceMappingGroups(s.ctx, &resourcemapping.ListResourceMappingGroupsRequest{
+		NamespaceId: scenarioDotComRmGroup.NamespaceID,
+	})
+	s.Require().NoError(err)
+	s.NotNil(rmGroups)
+	s.Len(rmGroups, 1)
+	s.Equal(scenarioDotComRmGroup.ID, rmGroups[0].GetId())
+	s.Equal(scenarioDotComRmGroup.NamespaceID, rmGroups[0].GetNamespaceId())
+	s.Equal(scenarioDotComRmGroup.Name, rmGroups[0].GetName())
+}
+
+func (s *ResourceMappingsSuite) Test_GetResourceMappingGroup() {
+	testRmGroup := s.f.GetResourceMappingGroupKey("example.com_ns_group_1")
+	rmGroup, err := s.db.PolicyClient.GetResourceMappingGroup(s.ctx, testRmGroup.ID)
+	s.Require().NoError(err)
+	s.NotNil(rmGroup)
+	s.Equal(testRmGroup.ID, rmGroup.GetId())
+	s.Equal(testRmGroup.NamespaceID, rmGroup.GetNamespaceId())
+	s.Equal(testRmGroup.Name, rmGroup.GetName())
+}
+
+func (s *ResourceMappingsSuite) Test_GetResourceMappingGroupWithUnknownIdFails() {
+	rmGroup, err := s.db.PolicyClient.GetResourceMappingGroup(s.ctx, unknownResourceMappingGroupID)
+	s.Require().ErrorIs(err, db.ErrNotFound)
+	s.Nil(rmGroup)
+}
+
+func (s *ResourceMappingsSuite) Test_CreateResourceMappingGroup() {
+	req := &resourcemapping.CreateResourceMappingGroupRequest{
+		NamespaceId: s.getExampleDotComNamespace().ID,
+		Name:        "example.com_ns_new_group",
+	}
+	rmGroup, err := s.db.PolicyClient.CreateResourceMappingGroup(s.ctx, req)
+	s.Require().NoError(err)
+	s.NotNil(rmGroup)
+}
+
+func (s *ResourceMappingsSuite) Test_CreateResourceMappingGroupWithUnknownNamespaceIdFails() {
```

Comment:
Not a requirement to merge, but it would be helpful to make these test names a little more readable with the format:
`Test_<function>_Scenario_Outcome` i.e. `Test_CreateResourceMappingGroup_WithUnknownNamespaceId_Fails`

Reply:
I actually wanted to do that but was following the format of the existing tests.  Totally fine with me and we could follow up and improve in a future PR.